CONTRIBUTING.md

```diff
@@ -16,6 +16,10 @@ limitations under the License.
 
 ## Contributing to AMP HTML
 
+The AMP HTML project strongly encourages technical contributions!
```

Comment:
Do you think it makes sense to link to https://www.ampproject.org/contribute/ or would that we redundant?